src/tsickle.ts

```diff
@@ -888,6 +888,36 @@ class ExternsWriter extends ClosureRewriter {
       this.writeExternsFunction(name.getText(), paramNames, namespace);
     }
 
+    // Handle method declarations/signatures first, since we need to deal with overloads.
+    // TODO: Build a collection of the remaining members
+    let methods: {[id: string]: ts.MethodDeclaration[];} = {};
+    for (let member of decl.members) {
+      if (member.kind === ts.SyntaxKind.MethodSignature ||
+          member.kind === ts.SyntaxKind.MethodDeclaration) {
```

Comment:
Is it worth handling the "else" case, if only to report an issue?  Sometimes we use switch statements to make it clear what the default case is.  There's `this.debugWarn()` which is intended for tsickle-internal TODOs (it's never shown to users of tsickle).